README.md

```diff
@@ -212,9 +218,25 @@ const contract = new HRC20('0x...00', ABI, wallet, {
 
 ### Methods
 
-#### balanceOf
+#### totalSupply(): Promise&lt;BN&gt;
```

Comment:
Review all functions definitions as they could have an optional parameter for gas options

Reply:
done here https://github.com/blockcoders/harmony-marketplace-sdk/pull/54/commits/c207a50c89bf3ef8378c17082fadab2286dbc4b9

let us know if there is any other comment or feedback ✌️